### PR TITLE
Hide Changelog Updates Behind Feature Flag

### PR DESCRIPTION
**Pull request by SudharakaP**
This hides the changelog updates behind a feature flag as discussed here; https://opencollective.slack.com/archives/C024M464XEV/p1623692927000200 🐨

### CODE REVIEW

File: components/TopBarProfileMenu.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ import StyledLink from './StyledLink';
 import { P, Span } from './Text';
 import { withUser } from './UserProvider';
 
+const CHANGE_LOG_UPDATES_ENABLED = process.env.CHANGE_LOG_UPDATES_ENABLED;
```
Comment (znarf) [inline components/TopBarProfileMenu.js:25]: Please parse with `parseToBoolean`

Comment (SudharakaP) [reply components/TopBarProfileMenu.js]: Thanks. done. 👍🏽 🐨

**Review by znarf — APPROVED** [review]
Ship it!

**Comment by znarf** [comment]
@SudharakaP This should have been deployed yesterday when it was merged.

**Comment by SudharakaP** [comment]
> @SudharakaP This should have been deployed yesterday when it was merged. This is a broken experience in production right now.

@znarf : Ops, I'll deploy it after standup. 😢

**Pull request merged**